### PR TITLE
Possibility to Specify the Version of a Stack

### PR DESCRIPTION
**Pull request by jmcs**

### CODE REVIEW

File: lizzy/api.py [hunk 1]
```diff
@@ -68,7 +70,8 @@ def create_stack(new_stack: dict) -> dict:
     keep_stacks = new_stack['keep_stacks']  # type: int
     new_traffic = new_stack['new_traffic']  # type: int
     image_version = new_stack['image_version']  # type: str
-    application_version = new_stack.get('application_version')  # type Optional[str]
+    application_version = new_stack.get('application_version')  # type: Optional[str]
+    stack_version = new_stack.get('stack_version')  # type: Optional[str]
```
Comment (rafaelcaricio) [inline lizzy/api.py]: Do you run any static type checking in lizzy source code so that is why you put those comments?

Comment (jmcs) [inline lizzy/api.py]: Currently I'm not using any static type checker, but PyCharm does type checking (and provides better auto completion) if the typing information is available.

Comment (rafaelcaricio) [inline lizzy/api.py]: :+1:

**Comment by rafaelcaricio** [comment]
:+1:

**Pull request merged**